src/index.html

```diff
@@ -20,8 +20,348 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO LOGO"
+          class="header__logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add the blank line between the links 
```suggestion
          </li>
          
          <li class="nav__item">
```